src/platform/intel/ace/lib/clk.c

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2022 Intel Corporation. All rights reserved.
+//
+// Author: Janusz Jankowski <janusz.jankowski@linux.intel.com>
+
+#include <sof/lib/notifier.h>
+#include <rtos/clk.h>
+#include <sof/drivers/ssp.h>
+#include <sof/sof.h>
+#include <sof/common.h>
+#include <sof/lib/memory.h>
+#include <rtos/spinlock.h>
+
+static SHARED_DATA struct clock_info platform_clocks_info[NUM_CLOCKS];
+
+void platform_clock_init(struct sof *sof)
```

Comment:
why we need this file?

Reply:
It is used by clock_get_freq function. For example called by basefw_config function from base_fw module. Zepphyr wrapper also using it in platform_dai_timestamp function.